src/components/Dialogs/__tests__/__snapshots__/MoreAbout.spec.jsx.snap

```diff
@@ -27,6 +27,7 @@ exports[`component::MoreAbout initial state renders without crashing 1`] = `
       We only publish the description of what happened if the consumer agrees to share it and after we take steps to 
       <a
         href="http://files.consumerfinance.gov/a/assets/201503_cfpb_Narrative-Scrubbing-Standard.pdf"
```

Comment:
i think we should update all of these hrefs to point to https.  I believe we've had issues with this before where the target server didn't automatically redirect to the https links.

Reply:
https://github.com/cfpb/ccdb5-ui/pull/382